qutebrowser/javascript/greasemonkey_wrapper.js

```diff
@@ -100,11 +100,8 @@
 
         const head = document.getElementsByTagName("head")[0];
```

Comment:
Not part of this PR, but can't

```
document.getElementsByTagName("head")[0];
```
be replaced with

```
document.head;
```

I could be understanding it wrong though...

Reply:
Eh, I just inherited that bit, and I am pretty sure it has been copied from somewhere. There is a similar greasemonkey shim in the chromium source somewhere.

>maybe it's coming from a child frame
Yeah I saw it being called twice for a `/post2` or some such url, but not the top level one.